activation/activationdb_test.go

```diff
@@ -26,7 +26,7 @@ func createLayerWithAtx(t *testing.T, msh *mesh.Mesh, id types.LayerID, numOfBlo
 			block1.AtxIds = append(block1.AtxIds, atx.Id())
 		}
 		block1.ViewEdges = append(block1.ViewEdges, views...)
-		err := msh.AddBlockWithTxs(block1, []*types.SerializableTransaction{}, atxs)
+		err := msh.AddBlockWithTxs(block1, []*types.AddressableSignedTransaction{}, atxs)
```

Comment:
why renaming this again? AddressableSignedTransaction is still SerializableTransaction...

Reply:
We can use one type with XDR (and having addr as an optional field). I'll open a technical debt.